Beginner_Projects/Currency-Converter-with toggletheme/currency_converter.py

```
@@ -0,0 +1,173 @@
+    #api url
+    url = f"https://api.currencyfreaks.com/latest?apikey={API_KEY}&symbols={from_curr},{to_curr}"
+
+    try:
+        response = requests.get(url)
+        response.raise_for_status()
+        data = response.json()
+
+        base_currency = data.get("base", "")
+        rates = data.get("rates",{})
+
+        #taking care of base case and falling back to usd incase either from or t_curr is the base currrency.
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)